mantidimaging/core/parallel/__init__.py

```diff
@@ -1,21 +1,31 @@
 from __future__ import absolute_import, division, print_function
 
+import sys
+
 __doc__ = """
```

Comment:
Can we not just put this at the top of the module has normal?

Reply:
`__future__` imports are supposed to come at the start of the file.

Reply:
There are a few things that are allowed to come before them though: https://docs.python.org/2/reference/simple_stmts.html#future.

We generally have them as the first thing after the docstring in all of our mantid code and I think it's cleaner. What do you think?